pkgs/applications/networking/nextcloud-client/default.nix

```diff
@@ -5,15 +5,19 @@
 
 stdenv.mkDerivation rec {
```

Comment:
Qt apps should use `mkDerivation` (it's qt5's) and you don't need to add `wrapQtAppsHook` since it does this automatically.